source/administration/config-settings.rst

```diff
@@ -2294,11 +2294,13 @@ The number of seconds to wait for a response from the database after opening a c
 
 At Rest Encrypt Key
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-32-character (to be randomly generated via Admin Console) salt available to encrypt and decrypt sensitive fields in database.
+A 32-character salt for encrypting and decrypting sensitive fields in the database. You can generate your own cryptographically random alphanumeric string, or you can go to **System Console > Advanced > Database** and click **Regenerate**, which displays the value until you click **Save**.
```

Comment:
We should remove the word salt here.  Technically it's a key not a salt.  In all the other places it is a salt (this one is special).